extensions/interactions/base_test.py

```diff
@@ -355,80 +355,75 @@ def test_default_interactions_are_valid(self):
         all_interaction_ids = (
             interaction_registry.Registry.get_all_interaction_ids())
         for interaction_id in all_interaction_ids:
+            # TODO(#10212):Ignoring the RatioExpressionInput for now as
+            # there's no learner's flow for the interaction yet.
+            # Once added this needs to be removed.
+
             # Check that the interaction id is valid.
             self.assertTrue(self._is_camel_cased(interaction_id))
             hyphenated_interaction_id = (
                 utils.camelcase_to_hyphenated(interaction_id))
-
```

Comment:
@aggarwalpulkit596 Could you please not delete all these newlines? They are here intentionally to break up the test into segments. Leave them as they were.

Reply:
Yes i would be reverting them